server/cyf.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE IF EXISTS energizers CASCADE;
+DROP TABLE IF EXISTS raitings CASCADE;
```

Comment:
Spelling is "ratings", also I suggest using the name `energizer_ratings` since it's a sub-entity to `energizer`

Reply:
Thank you for spotting this, good suggestion